src/Bicep.LangServer/Handlers/BicepDeploymentStartCommandHandler.cs

```diff
@@ -101,5 +147,20 @@ private void PostDeployStartResultTelemetryEvent(string deployId, bool isSuccess
 
             telemetryProvider.PostEvent(telemetryEvent);
         }
+
+        private string GetParamsJsonOutput(Compilation compilation)
+        {
+            var paramsModel = compilation.GetEntrypointSemanticModel();
+
+            var paramsOutputBuffer = new StringBuilder();
+            using var paramsOutputWriter = new StringWriter(paramsOutputBuffer);
+
+            var paramsEmitter = new ParametersEmitter(paramsModel);
+            var paramsResult = paramsEmitter.Emit(paramsOutputWriter);
```

Comment:
`paramsResult` will contain info on whether the compilation succeeded or failed in the `Status` property. Can we use this, rather than doing our own check for error diagnostics?